datasm/resources/datasm_config.yaml

```diff
@@ -1,3 +1,6 @@
+### WARNING - DO NOT EDIT DIRECTLY - changes can be overwritten.    ###
+### USE (resource)/datasm_config.template.sh to modity and recreate ###
```

Comment:
This file gets edited through `(resource)/datasm_config.template.sh`?

Reply:
Yes, it gets "built", because the top-level paths are defined by an external OS call to a file the user can manipulate (path values, not the path keys) without having to build/reinstall datasm.  They would still need to rerun datasm_config.template.sh

Reply:
@tomvothecoder In truth, the path-part of that config (at the top) should eventually be removed (they inherit from the external dsm_paths config in the first place) and could then become a static config once again.   Another day.